src/App.tsx

```diff
@@ -1,24 +1,239 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useState,
+  useEffect,
+  useMemo,
+  useCallback,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import {
+  getTodos,
+  createTodo,
+  deleteTodo,
+  toogleTodo,
+  updateTodo,
+} from './api/todos';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import {
+  TodoList,
+  Header,
+  Footer,
+  Notification,
+} from './components';
+import { warningTimer } from './utils/warningTimer';
+import { FilterTodos } from './types/FilterTodos';
+
+const USER_ID = 6336;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [creatingTodo, setCreatingTodo] = useState<Todo | null>(null);
+  const [todosInProcessed, setTodosInProcessed] = useState<Todo[]>([]);
+  const [title, setTitle] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [selectFilter, setSelectFilter] = useState<FilterTodos>(FilterTodos.ALL);
+
+  const activeTodos = todos.filter(({ completed }) => !completed);
+
+  const pushError = (message: string) => {
+    setErrorMessage(message);
+    warningTimer(setErrorMessage, '', 3000);
+  };
+
+  const handleEventChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.target;
+
+    setTitle(value);
+  };
+
+  const handleAddTodo = useCallback(async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    if (!title.trim().length) {
+      pushError('Title can\'t be empty');
+      setTitle('');
+
+      return;
+    }
+
+    const newTodo = {
+      id: 0,
+      title,
+      userId: USER_ID,
+      completed: false,
+    };
+
+    try {
```

Comment:
what if error will appear in code above try block? It wouldn't be handled  and app will be crushed
```suggestion
  const handleAddTodo = useCallback(async (e: React.FormEvent<HTMLFormElement>) => {
    try {
      e.preventDefault();

      if (!title.trim().length) {
        pushError('Title can\'t be empty');
        setTitle('');

        return;
      }

      const newTodo = {
        id: 0,
        title,
        userId: USER_ID,
        completed: false,
      };

```